static/src/js/pages/KeywordManagerPage/KeywordManagerPage.jsx

```diff
@@ -300,6 +346,58 @@ const KeywordManagerPage = () => {
         message="You are now viewing the live published keyword version. Changes made to this version will show up on the website right away."
         actions={warningModalActions}
       />
+      <CustomModal
+        show={showPublishModal}
+        toggleModal={() => setShowPublishModal(false)}
```

Comment:
```suggestion
        toggleModal={setShowPublishModal(false)}
```
I don't think we need to define these in the render each time

Reply:
call setPublishSuccess directly calls redrawing in an infinite loop